plugin/src/main/groovy/com/novoda/staticanalysis/EvaluateViolationsTask.groovy

```diff
@@ -1,14 +1,15 @@
 package com.novoda.staticanalysis
 
 import com.novoda.staticanalysis.internal.Violations
-import org.gradle.api.DefaultTask
 import org.gradle.api.GradleException
 import org.gradle.api.NamedDomainObjectContainer
+import org.gradle.api.internal.ConventionTask
 import org.gradle.api.tasks.TaskAction
 
-class EvaluateViolationsTask extends DefaultTask {
+class EvaluateViolationsTask extends ConventionTask {
```

Comment:
Is this really necessary? We used conventionMapping with extending `DefaultTask`
I wanted to comment because it is in `internal` package.